add-scene-layer-with-elevation/src/main/java/com/esri/arcgismaps/sample/addscenelayerwithelevation/screens/MainScreen.kt

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2023 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.addscenelayerwithelevation.screens
+
+import android.app.Application
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.esri.arcgismaps.sample.addscenelayerwithelevation.components.ComposeSceneView
+import com.esri.arcgismaps.sample.addscenelayerwithelevation.components.SceneViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen layout for the sample app
+ */
+@Composable
+fun MainScreen(sampleName: String, application: Application) {
+    // create a ViewModel to handle MapView interactions
+    val sceneViewModel = SceneViewModel(application)
+
+    Scaffold(
+        topBar = { SampleTopAppBar(title = sampleName) },
+        content = {
+            Column(
+                modifier = Modifier
+                    .fillMaxSize()
+                    .padding(it)
+            ) {
+                // composable function that wraps the MapView
+                ComposeSceneView(
+                    modifier = Modifier
+                        .fillMaxSize(),
+                    sceneViewModel = sceneViewModel,
+                    onSingleTap = {
+                        sceneViewModel.addSceneLayer()
+                    }
```

Comment:
Could the `SceneLayer` be loaded as part of the `init` of the `SceneViewModel`? So it could be loaded on sample launch: 
```kotlin
// add a scene layer to the ArcGISScene
init {
    // create a scene layer from a scene service for viewing buildings
    val sceneLayer = ArcGISSceneLayer(application.getString(R.string.brest_buildings))
    // add the scene layer to the scene's operational layer
    sceneViewState.arcGISScene.operationalLayers.add(sceneLayer)
}
```

Reply:
Yeah, I was meaning to do this, when I was reviewing one of your samples, good suggestion!